draft-mglt-ipsecme-diet-esp.xml

```diff
@@ -211,45 +211,57 @@ and up to 66 bytes for VPN set over a single TCP or UDP session.</t>
-<t>ESP Header Compression compresses the standard ESP payload by
-compressing different fields with specific compression rules performed
-in the ESP stack. Concerned fields include those of the ESP protocol, as
+
+<t>EHC is able to compress any protocol encapsulated in ESP and ESP itself.
+Concerned fields include those of the ESP protocol, as
 well as other protocols in the ESP payload such as the IP header when
-the tunnel mode is used, the UDP or the TCP header. In fact non ESP
-fields may be compressed by ESP under certain circumstances and ESP
-Header Compression is not intended to provide a generic way outside of
-ESP to compress these protocols. Further compression of the ESP payload
-may be performed by generic mechanism and outside ESP with more generic
-mechanisms such as for example ROHCoverIPsec <xref target="RFC5858"/> or
-SCHC <xref target="I-D.toutain-6lpwa-ipv6-static-context-hc"/> which are
-orthogonal to ESP Header Compression.</t>
- 
-<t>As depicted in <xref target="fig:overview"/>, in order to compress
-the ESP packets, the two peers are expected to agree on the EHC Strategy
-- Diet-ESP in our case - as well as some extra parameters needed to
-derive the EHC Rules and EHC Context.</t>
- 
+the tunnel mode is used, but also upper layer protocols, such as
+the UDP or the TCP header. Non ESP
+fields may be compressed by ESP under certain circumstances, but EHC
+is not intended to provide a generic way outside of
+ESP to compress these protocols.
+Compression of the unprotected IP header and the unencrypted ESP header
+may be performed by mechanism such as 6LoWPAN <xref target="RFC4994"/>,
+SCHC <xref target="I-D.toutain-6lpwa-ipv6-static-context-hc"/>,
+ROHC <xref target="RFC5795"/> or 6LoWPAN-GHC <xref target="RFC7400"/>.
+</t>
+
+<t>EHC is based on a static compression context, EHC Rules coordinated
+by an EHC Strategy:
+  <list style="hanging" hangIndent="4">
+      <t hangText="EHC Context:">Stores the information of a specific
+      header field which can be compressed by EHC. This can be specific
+      header values such as IP addresses or L4 ports do not have to be
+      send on the wire at all, or compression information for fields
+      which can be partially compressed, such as sequence numbers.</t>
+      <t hangText="EHC Rules:">Defines how the information of the EHC
+      Context is used to compress a specific field. It defines
+      compression functions, such as "elided", "least significant
+      byte" and others, being applied on the header field.</t>
+      <t hangText="EHC Strategy:">Is applied to efficiently
+      coordinate EHC Context and EHC Strategy. The EHC Strategy
+      "Diet-ESP" defined in this document utilizes the information
+      in the IPsec SA to pre-define the EHC Context without
+      explicitly exchanging the EHC Context. </t>
+  </list>
+</t>
+
+
+<t>As depicted in <xref target="fig:overview"/>, the EHC Strategy
+- Diet-ESP in our case - as well as EHC Context is agreed between

```

Comment:
minor: I would rephrase this as `the EHC Strategy - Diet-ESP in our case - and the EHC Context are agreed upon between the two peers`